lib/Driver/ToolChains.cpp

```diff
@@ -1016,6 +1016,17 @@ static void getRuntimeLibraryPath(SmallVectorImpl<char> &runtimeLibPath,
                           getPlatformNameForTriple(TC.getTriple()));
 }
 
+/// Get the runtime library link path with the target triple's specific arch
+/// library folder.
+static void getRuntimeLibraryPathWithArch(SmallVectorImpl<char> &runtimeLibPath,
```

Comment:
Can you call this `getRuntimeSharedLibraryPathWithArch` to match `getRuntimeStaticLibraryPathWithArch`?